### PR TITLE
Fix libdrm symbol errors on Ubuntu 16.04

### PR DESCRIPTION
**Pull request by ljbade**
@jck Not sure how to run this to test it

### CODE REVIEW

File: scripts/build_release.py [hunk 1]
```diff
@@ -39,6 +39,9 @@ def build_linux():
     maybe_remove(out)
     shutil.move(out_pyi, out)
 
+    # prevents missing export error on ubuntu 16.04 with intel graphics
+    maybe_remove(os.path.join(out, 'libdrm.so.2'))
```
Comment (jck) [inline scripts/build_release.py]: looks like you'll need to update the maybe_remove function to let it delete files in addition to directories.

Comment (ljbade) [reply scripts/build_release.py]: bah, do you have some code that will work?

**Comment by ljbade** [comment]
@jck also how do i build the full .tar.gz, or does travis build it for me?

File: scripts/build_release.py (hunk 1 above)
Comment (ljbade) [reply scripts/build_release.py]: nvm, I just changed it to `os.remove` since it should always exist anyway

**Comment by ljbade** [comment]
@jck where can i find the .tar.gz Travis built?

**Pull request merged**